themes/godotengine/pages/governance.htm

```diff
@@ -0,0 +1,84 @@
+    project.
+    <strong>As no company is behind Godot, it can't be sold or purchased.</strong>
+  </p>
+  <p>
+    Within the organization sits the Godot
+    <a href="/contact#plc">Project Leadership Comittee</a>. This committee
+    decides and authorizes the usage of project funds (donations and grants).
+  </p>
+  <p>
+    <a href="/license">Godot's license is MIT</a>. This means you can do pretty
+    much everything with it, as long as you don't misrepresent the origin of the
+    software. In practice, even with that condition in place, you can do
+    anything you want as if you wrote Godot yourself.
+  </p>
+  <p>
+    Still, the MIT license allows creating proprietary derivatives.
+    Many companies have private forks of the engine to suit their own needs, but
+    what is open will remain open forever. <strong>No legal way exists to cease
+    your right to use Godot.</strong> It's as if it was your own property;
+    nobody can take it from you.
+  </p>
+  <p>
+    Godot is made by thousands of skilled contributors who donate their free
+    time (some of them are paid from donations). Even if some contributors
+    suddenly decided to push new Godot versions as proprietary, it would be
+    difficult for them to compete or be as productive as the rest. Godot is a
+    true community effort, created and owned by everyone.
+  </p>
+
+  <h3>Project leadership, fund usage and decisions</h3>
+  <p>
+    While Godot has a <a href="/contact#plc">Project Leadership Committee</a>,
+    <strong>there is no central authority deciding what gets done</strong>.
+    The Project Leadership Committee rarely decides something on its own.
+    Usage of funds is discussed and approved by the
+    <em><a href="https://github.com/orgs/godotengine/people">advisors</a></em>,
+    which is a group of about 30 veteran trusted contributors and area
+    maintainers. We hold online meetings and discuss these topics regularly.
+  </p>
+  <p>
+    Godot is a large project and the advisors have a great reach to all
+    areas of the project, both technical and social. Our meetings are generally
+    well-informed, which allows steering the project direction more precisely.
+  </p>
+  <p>
+    Contributors cannot request to join the advisors or the Project
+    Leadership Committee. Instead, contributions in Godot are based on merit. If
+    we agree that a pull request is good, then we'll merge it. However,
+    leadership is based on trust. Contributors who are deemed trustworthy enough
+    can be asked to join the advisors or the Project Leadership
+    Committee.
+  </p>
+  <p>
+    Therefore, the common way to join the advisors group is to take
+    maintainership of an engine area or contribute significantly to it. Show
+    other maintainers that you can be trusted over time by fixing bugs and
+    reviewing pull requests. This will give you a stronger voice on deciding
+    where the project wants to go.
```

Comment:
Not sure if this should be included. As Akien said people might think that joining the advisors is the ultimate goal but that should not be the case. We might have trusted contributors who won't become advisors.

Reply:
This is also not resolved. Did you forget to push changes?